src/main/java/tests/FruitTransactionTest.java

```diff
@@ -0,0 +1,21 @@
+package tests;
+
+import core.basesyntax.*;
```

Comment:
According to the checklist, you should avoid keeping all logic in a single package. It's recommended to organize your test classes into corresponding packages, such as `model` for testing domain objects like `FruitTransaction`.